protobufs/livekit_sip.proto

```diff
@@ -416,3 +417,23 @@ message TransferSIPParticipantRequest {
   string room_name = 2;
   string transfer_to = 3;
 }
+
+message SIPCallInfo {
+  string call_id = 1;
+  string trunk_id = 2;
+  string room_id = 3; // ID of the current/previous room published to
```

Comment:
should we include the name as well?

Note to @dennwc, room creation is async, and it could take a few seconds before the room ID resolves.